MyFiles/Class.cs

```diff
@@ -0,0 +1,63 @@
+﻿//Class.cs
+
+using System.ComponentModel.Design;
+
+namespace VendingMachineManager
+{
+    public class VendingMachine
+    {
+        private string _product;
+        private string[] Menu = new string[5];
```

Comment:
Typically having a collection type (string array) with a singular attribute name (`Menu`) is a code smell. I.e., is this a collection of menus? A collection of menu products? Consider renaming for clarity.